common/artifacts/909_wc_artifacts_library_librams.txt

```diff
@@ -312,12 +286,11 @@ libram_of_devoutness_02 = {
 	stacking = no
 	indestructible = yes
 
-	learning = 2
-	combat_rating = 2
+	learning = 1
 
-	monthly_character_piety = 0.2
+	monthly_character_piety = 0.3
+	light_group_opinion = 8
 
-	light_group_opinion = 4
 
 	picture = "GFX_libram_holiness"
 
```

Comment:
you said, playing by the rules would make balancing impossible... this one is good :)